packages/scripts/src/utils/evaluate-module.ts

```diff
@@ -9,6 +9,10 @@ export function evaluateModule(filePaths: string | string[]): Module {
     filePaths = typeof filePaths === 'string' ? [filePaths] : filePaths;
     const entryModule = new Module('entry-module');
     entryModule.filename = 'entry-module.js';
+    const resolutionPaths = require.resolve.paths(filePaths[0]);
```

Comment:
`// we want node to be able to resolve package requests. we use first module to calculate lookup locations.`